### PR TITLE
Logic added to the calculator

### PR DESCRIPTION
**Pull request by yahir91**
In this milestone, I added the logic to the calculator with two functions operate and calculate.

### CODE REVIEW

File: src/logic/operate.js [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = Big(numberOne);
+  const secondNumber = Big(numberTwo);
+
+  switch (operation) {
+    case '+':
+      return firstNumber + secondNumber;
+    case '-':
+      return firstNumber - secondNumber;
+    case 'X':
```
Comment (bellom) [inline src/logic/operate.js]: You might wanna use lowercase `x` to match the size with the rest of the operator.

File: src/logic/operate.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+      return firstNumber * secondNumber;
+    case '÷':
+      if (secondNumber === '0') {
+        return '0';
+      }
+      return firstNumber / secondNumber;
+    case '%':
+      return firstNumber * Big(0.01);
+    case '+/-':
+      return firstNumber * Big(-1);
```
Comment (bellom) [inline src/logic/operate.js]: Job well done but you don't have to do the calculating yourself, that where the use of `big` comes in -> [big js library](https://github.com/favio41/big-js#readme)
example: case division will be -> `firstNumber.minus(secondNumber)`

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @yahir91  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best,**

**Oluwaseun Bello**
**_Slack Handle - @sean bellom_**

**Comment by bellom** [comment]
@yahir91, Noted
This CRR, has been mistakenly marked as approved.
I already reach out to the dev guys to revert it.

Sorry for this.

**Comment by yahir91** [comment]
Thank you for the review, x has been changed to lowercase, and the Big.js methods have been used properly

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,30 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const firstNumber = new Big(numberOne);
+  const secondNumber = numberTwo;
+
```
Comment (uche-inyama) [inline src/logic/operate.js]: You should also create a new instance of Big for the secondNumber. 

```
    const secondNumber = new Big(numberTwo)
````

**Review by uche-inyama — CHANGES_REQUESTED** [review]
## Changes Required

### Hi @yahir91 :wave: 

You are almost done. I have a left of comment below, update operate.js accordingly.

#### That's it, make the change and submit another form for a code review.


### Have a great day :slightly_smiling_face: 


#### ``` Uchechukwu Henry Inyama ```

**Review by arslanbisharat — APPROVED** [review]
# Approved 🥇 

Thank you for submitting this project. You have done a great job!

- [x] Your commit messages are very descriptive too. I am very happy to see that.  👏 
- [x] No Linter error found. 👏 
- [x] Met Minimum requirements to pass the project.  

I am very happy to review your project. Looking forward to amazing new projects from your side. 

**Best,
Arslan Bisharat**
_You can always contact me on slack at @arslanbisharat 📞_

**Pull request closed without merging**